packages/utils/src/object.ts

```diff
@@ -135,47 +124,38 @@ export function getWalkSource(value: any): {
 
     const event = value as unknown as SimpleEvent;
```

Comment:
I think we don't need this cast anymore.

Reply:
You're right. I forget why it was there in the first place. Anyway, removed.